pkg/test/framework/components/bookinfo/configs.go

```diff
@@ -137,6 +137,17 @@ func GetDestinationRuleConfigFile(t test.Failer, ctx resource.Context) ConfigFil
 	return NetworkingDestinationRuleAll
 }
 
+func GetDestinationRuleConfigFileWithoutFail(ctx resource.Context) (ConfigFile, error) {
```

Comment:
The correct naming for this function and refactoring around it is in PR: https://github.com/istio/istio/pull/14769/files#diff-10f04942d39e462334875e1902daf6b8